bib/cmd/bootc-image-builder/main.go

```diff
@@ -144,7 +144,7 @@ func build(cmd *cobra.Command, args []string) {
 	upload := false
 	if region, _ := cmd.Flags().GetString("aws-region"); region != "" {
 		if imgType != "ami" {
-			fail("aws flags set for non-ami image type")
+			fail(fmt.Sprintf("aws flags set for non-ami image type (type is set to %s)", imgType))
```

Comment:
Can you make this change in a separate commit so it's not mixed in with the other change?

Also the commit message subject line is a bit too long (should be < 72 chars).

I think you can add the Jira ticket ID in the message body and it will still work.